packages/neon_framework/lib/src/utils/image_utils.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:xml/xml.dart';
+import 'package:xml/xpath.dart';
+
+/// Utilities for working with images.
+class ImageUtils {
+  ImageUtils._();
+
+  /// Rewrites percent dimensions in SVGs to be absolute.
+  ///
+  /// This is required because the `flutter_svg` package can not handle percent dimensions.
```

Comment:
Is there an upstream issue we can link to?

Reply:
There are many open issues about that, I doubt it makes sense to link to them